src/hooks/useRdfCollection.js

```diff
@@ -0,0 +1,84 @@
+import { useQuery, useMutation, useQueryClient } from '@tanstack/react-query';
+import { useState } from 'react';
+import {
+  getThing,
+  removeThing,
+  setThing,
+  createSolidDataset,
+  getSolidDataset,
+  saveSolidDatasetAt
+} from '@inrupt/solid-client';
+
+const useRdfCollection = (parse, serialize, fileUrl, fetchData) => {
```

Comment:
Think we could include JSDoc to describe what the new parameters are for this shared hook. The description in the PR should suffice.

Reply:
Sounds good. Added.